src/styles/styles.js

```diff
@@ -2878,6 +2878,48 @@ const styles = {
         paddingTop: 80,
         paddingBottom: 45,
     },
+
+    footerWrapper: {
+        backgroundColor: themeColors.midtone,
+        fontSize: variables.fontSizeNormal,
+        paddingTop: 64,
+        paddingLeft: 17,
+    },
+
+    footerColumnsContainer: {
+        maxWidth: 1100,
+        flex: 1,
+        flexWrap: 'wrap',
+        marginBottom: 40,
+    },
+
+    footerColumn: {
+        padding: 15,
+    },
+
+    footerTitle: {
+        fontSize: variables.fontSizeLarge,
```

Comment:
Maybe we can use the same font size as page headers, whatever they are. So maybe this is fine as-is but curious what its value is?

Reply:
It looks like page headers are 17px (same as fontSizeLarge). There's not a ton of font styles set up using new kansas which is why I added it manually, but I could do it a different way if that makes more sense

Reply:
perfect, let's do 17px.